### PR TITLE
fix:Add bookkeeping task for CA post calls.

### PR DESCRIPTION
**Pull request by rafiu007**
This bookkeeping task will store data into RDS from CA post calls.

### CODE REVIEW

File: bayesian/utils.py [hunk 1]
```diff
@@ -103,6 +103,20 @@ def server_create_component_bookkeeping(ecosystem, name, version, user_profile):
     return server_run_flow('componentApiFlow', args)
 
 
+def create_component_bookkeeping(ecosystem, packages_list, user_id, user_agent):
+    """Run the component analysis for given ecosystem+package+version."""
+    args = {
+        'api_name': 'component_analyses_post',
+        'external_request_id': uuid.uuid4().hex,
```
Comment (prashbnair) [inline bayesian/utils.py:110]: My suggestion will be to generate this on the client side and use this for all the POST requests for a single stack. With this we will be able to figure out the stack.

Are we not populating the manifest_hash

File: bayesian/api/api_v2.py [hunk 2]
```diff
@@ -39,6 +39,7 @@
     get_known_unknown_pkgs, add_unknown_pkg_info, get_batch_ca_data
 from bayesian.utils import (get_system_version,
                             server_create_component_bookkeeping,
```
Comment (prashbnair) [inline bayesian/api/api_v2.py:41]: Is this component being used?

**Review by prashbnair — CHANGES_REQUESTED** [review]

File: bayesian/api/api_v2.py (hunk 2 above)
Comment (rafiu007) [reply bayesian/api/api_v2.py:41]: when CA get call is made that is used to store data in api_requests table.

Comment (prashbnair) [reply bayesian/api/api_v2.py:41]: I dont think we need to track that anymore.

File: bayesian/utils.py (hunk 1 above)
Comment (rafiu007) [reply bayesian/utils.py:110]: For the manifest_hash I will add codeto fetch it from the user.

File: bayesian/api/api_v2.py (hunk 2 above)
Comment (rafiu007) [reply bayesian/api/api_v2.py:41]: I will modify accordingly.

**Pull request closed without merging**